Cubical/Categories/Constructions/Free/CategoryWithTerminal.agda

```diff
@@ -0,0 +1,128 @@
+-- Free category with a terminal object, over a Quiver
+module Cubical.Categories.Constructions.Free.CategoryWithTerminal where
+
+open import Cubical.Foundations.Prelude
+
+open import Cubical.Categories.Category.Base
+open import Cubical.Categories.Limits.Terminal.More
+open import Cubical.Data.Quiver.Base
+open import Cubical.Data.Sum.Base as Sum hiding (elim)
+open import Cubical.Data.Unit
+open import Cubical.Categories.Displayed.Base
+open import Cubical.Categories.Presheaf
+open import Cubical.Categories.Displayed.Presheaf
+open import Cubical.Categories.Displayed.Limits.Terminal
+open import Cubical.Foundations.Equiv
+open import Cubical.Data.Sigma.Properties
+open import Cubical.Categories.Displayed.Section.Base
+
+private
+  variable
+    ℓg ℓg' ℓC ℓC' ℓCᴰ ℓCᴰ' : Level
+
+CategoryWithTerminal' : (ℓC ℓC' : Level) → Type _
+CategoryWithTerminal' ℓC ℓC' = Σ[ C ∈ Category ℓC ℓC' ] Terminal' C
+
+-- freely throw in a terminal object
+module _ (Ob : Type ℓg) where
+
+    -- adjoin a new terminal object
```

Comment:
please use 2 spaces for indentation, not 4.

Reply:
patched with https://github.com/maxsnew/cubical-categorical-logic/pull/79/commits/5f221ae2653f7408a1ff184dc7efc4f3fc02c924